bundle-tools.sh

```
@@ -0,0 +1,61 @@
+# Normalize version for make (operator-sdk expects semver without "v", e.g. 3.3.0).
+_normalize_version () { echo "${1#v}"; }
+
+# Build and push both operator image and bundle image for RHOAI (rhods-operator).
+# Usage: build-bundle <version> [operator_namespace]
+# Example: build-bundle 3.3.4   or   build-bundle v3.3.4
+# Uses ODH_PLATFORM_TYPE=rhoai so operator and bundle match downstream (rhoai.Dockerfile, rhods-operator).
+build-bundle () {
+  VERSION_RAW="${1:?Usage: build-bundle <version> [operator_namespace], e.g. build-bundle 3.3.4}"
+  VERSION="$(_normalize_version "$VERSION_RAW")"
+  OPERATOR_NAMESPACE="${2:-redhat-ods-operator}"
```

Shellcheck (0.11.0)

[warning] OPERATOR_NAMESPACE appears unused. Verify use (or export if used externally).

(SC2034)